Kiosk/Bid Fulfillment/StripeManager.swift

```diff
@@ -10,10 +10,10 @@ protocol Tokenable {
 extension STPToken: Tokenable { }
 
 protocol Clientable {
-    func createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?)
+    func kiosk_createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?)
 }
 extension STPAPIClient: Clientable {
-    func createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?) {
+    func kiosk_createToken(withCard card: STPCardParams, completion: ((Tokenable?, Error?) -> Void)?) {
```

Comment:
So naming the protocol method the same as the actual method caused some confusion, and an infinite recursion loop 😒